pandas/tests/io/json/test_normalize.py

```diff
@@ -631,6 +640,7 @@ def test_missing_meta(self, missing_metadata):
         ex_data = [
             [9562, "Morris St.", "Massillon", "OH", 44646, "Alice"],
             [8449, "Spring St.", "Elizabethton", "TN", 37643, np.nan],
+            [8449, np.nan, np.nan, np.nan, np.nan, "Minnie"],
```

Comment:
Can you also create a dedicated test with just an example of a valid metadata key whose object is an empty array? The way things have evolved in this PR this makes sense, but I don't believe the keyword argument of `errors="ignore"` has anything to do with the behavior here. That is documented as handling the presence/non-presence of they key, but we are here actually interested in the value

Reply:
please see new test_missing_nested_meta_traverse_empty_list